_pages/yaml-file-format.md

```diff
@@ -341,6 +341,17 @@ Labels are short to fit in forms and other constrained-space UI elements; for mo
     
     If `superstructures` is an empty `map`, then the structure is a record and must not appear in any other structure's `substructures`.
 
+-   <table><tbody>
+    <tr><th>Key</th><td><code>translated from</code></td></tr>
+    <tr><th>Type</th><td>Language Tag</td></tr>
+    <tr><th>Required by</th><td>—</td></tr>
+    <tr><th>Allowed by</th><td>all</td></tr>
+    </tbody></table>
+
+    A language tag, as specified in BCP 47, which must be different from the language tag included in the `lang` key.
+    If present, this field indicates that this YAML file is a translation of a different YAML file, one presented in the indicated language.
+    in the event that the two disagree, the other YAML file shall be taken as more correct.
```

Comment:
```suggestion
    In the event that the two disagree, the other YAML file shall be taken as more correct.
```

Do we also want a sentence that says that the other YAML file must be in the repository?